synapseutils/copy.py

```diff
@@ -389,7 +420,7 @@ def copyWiki(syn, entity, destinationId, entitySubPageId=None, destinationSubPag
         for oldWikiId in wikiIdMap.keys():
```

Comment:
What happens if you have booth syn123:syn9999 and syn1234:syn1111 in the mapping.  Won't it be cause syn1234 to become syn11114?